internal/prompts/descriptions/get_traces.md

```
@@ -155,31 +155,52 @@ Note that regexp parsing operators also work as regexp filters
-- **Timestamp**: Trace timestamp
-- **Duration**: Span duration
-- **attributes['field_name']**: Span attributes (OpenTelemetry semantic conventions)
-- **resources['field_name']**: Resource attributes (prefixed with `resource_`), extract field name by stripping resource_
+### ALWAYS call get_trace_attributes before filtering on resource or span attributes
 
-### Custom Fields for user's environment:
-In addition to standard labels, the list of available customer-specific attribute labels is below. In the query, the following rule should be applied to get the attribute from the field name - if the field matches the pattern with `resource_fieldname` the attribute is `resources['fieldname']`. Otherwise it is `attributes['fieldname']`.
-Any attribute used in the query should either be a standard attribute or available from get_trace_attributes
+`get_trace_attributes` returns a JSON array. Each entry has a `filter_field` — use it verbatim in filter conditions. Never transform or guess field syntax.
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)